app/assets/stylesheets/views/_consultation.scss

```diff
@@ -1,15 +0,0 @@
-.consultation {
-  @include sidebar-with-body;
-
-  .consultation-date {
-    @include govuk-font(19, $weight: bold);
-  }
-
-  .original-consultation {
-    border-top: 1px solid $govuk-border-colour;
-
-    @include govuk-media-query($from: tablet) {
-      padding-top: govuk-spacing(6);
```

Comment:
When testing locally, there is no white space at the top of the consultation component. I imagine we can fix this by adding a utility class to the HTML.

When testing on mobile, we still have white space at the top of the element, this is because [`mobile_top_margin` is set to true](https://github.com/alphagov/government-frontend/blob/54b3ea5fc89e07ae9401f81855f447060b52bebc/app/views/content_items/consultation.html.erb#L56).

So it appears we have 2 different approaches for adding some space to the top of the element, mobile is using margin-top and desktop is using padding-top.

Reply:
Thanks, good spot. Should now be fixed by adding `govuk-!-margin-top-6`.